pkg/sql/catalog/descpb/structured.go

```diff
@@ -391,3 +392,9 @@ func (ni NameInfo) GetParentSchemaID() ID {
 func (ni NameInfo) GetName() string {
 	return ni.Name
 }
+
+func init() {
+	protoreflect.RegisterShorthand("Descriptor", (*Descriptor)(nil))
+	protoreflect.RegisterShorthand("desc", (*Descriptor)(nil))
+	protoreflect.RegisterShorthand("TableDescriptor", (*TableDescriptor)(nil))
```

Comment:
Would `TableDescriptor` ever be used in practice? For what it's worth, I never have, it's always the `Descriptor` union type.

Reply:
Good point, removed it.